xbmc/FileItem.cpp

```diff
@@ -490,8 +476,7 @@ void CFileItem::Reset()
   m_epgInfoTag.reset();
   delete m_pvrChannelInfoTag;
   m_pvrChannelInfoTag=NULL;
-  delete m_pvrRecordingInfoTag;
-  m_pvrRecordingInfoTag=NULL;
+  m_pvrRecordingInfoTag.reset();
```

Comment:
Sorry, don't understand what you mean. What's the problem here?

BTW: Did it the same way for epginfotag and there it was kay for you. See some lines of code above...